peerdid/peer_did_utils.py

```diff
@@ -198,20 +270,22 @@ def _build_did_doc_numalgo_2(peer_did: PEER_DID) -> dict:
         else:
             service = key[1:]
     decoded_encnumbasises = [
-        _decode_encnumbasis(key, peer_did) for key in keys_without_purpose_code
+        _decode_encnumbasis(key, peer_did, format) for key in keys_without_purpose_code
     ]
     decoded_service = _decode_service(service, peer_did)
 
     authentication = []
     key_agreement = []
     for i in range(len(decoded_encnumbasises)):
         if (
-            decoded_encnumbasises[i]["type"] in PublicKeyTypeAuthentication.__members__
+            decoded_encnumbasises[i]["type"]
+            in VerificationMaterialTypeAuthentication.values()
             and keys[i][0] == "V"
```

Comment:
out of the current changes but that letter `"V"` looks like a hard-coded magic

Reply:
Refactored the code to avoid hard-coded values.